Source/path.cpp

```diff
@@ -11,116 +11,143 @@
 #include "objects.h"
 
 namespace devilution {
-
 namespace {
 
+constexpr size_t MaxPathNodes = 300;
+
+struct PathNode {
+	static constexpr uint16_t InvalidIndex = std::numeric_limits<uint16_t>::max();

```

Comment:
Is it worth using `uint8_t` as an index type here? values currently will always be in the range 0-24

```suggestion
	using IndexType = uint_fast8_t
	static constexpr IndexType InvalidIndex = std::numeric_limits<IndexType>::max()
```

Reply:
`MaxPathNodes` is 300, are you saying we can reduce `MaxPathNodes` to 24?

Reply:
My mistake, I missed the define and use of MaxPathNodes😂